### PR TITLE
Feature/orcasound dataset

### PR DESCRIPTION
**Pull request by mosheman5**
- insert OrcaSound dataset support to our framework
- Add option for train/val splits in metadata, less csv files needed
- remove old unneeded functions

### CODE REVIEW

**Comment by mosheman5** [comment]
Trained model:
https://wandb.ai/deep_voice/finding_willy/runs/329v4rka/overview?workspace=user-deep_voice
validated performance on sparse inspection of test dataset (TODO adapt test dataset metadata and check all the files)

File: src/conf/data/orcasound.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    data_sample_rate: ${data.data_sample_rate}
+    sample_rate: ${data.sample_rate}
+    equalize_data: true
+    slice_flag: false
+    metadata_by_split: yes
+    margin_ratio: 0
+  val_dataset:
+    _target_: data.ClassifierDataset
+    data_path: '../datasets/orcasound/train_data/wav'
+    mode: val
```
Comment (noamzbr) [inline src/conf/data/orcasound.yaml:26]: Think this should have a more specific name (e.g. "split_label" or "split_tag" or such)

File: src/conf/data/orcasound.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    augmentations_p: 0
+    augmentations: null
+    preprocessors: ${_preprocessors}
+    seq_length: 1
+    len_buffer: 0.1
+    data_sample_rate: ${data.data_sample_rate}
+    sample_rate: ${data.sample_rate}
+    equalize_data: false
+    slice_flag: true
+    metadata_by_split: yes
```
Comment (noamzbr) [inline src/conf/data/orcasound.yaml:37]: Think this should be "split_by_metadata"

File: src/utils.py [hunk 3]
```diff
@@ -21,45 +21,6 @@
     collectionsAbc = collections
 
 
-def load_audio(filepath: str, sr: int, max_val=0.9):
```
Comment (noamzbr) [inline src/utils.py:24]: I trust we're not using that anymore?

**Review by noamzbr — COMMENTED** [review]
Didn't check the notebook yet but commented on the rest of the code

**Comment by noamzbr** [comment]
Notebook comments:

1. Making sure - the data for this data is at `s3://deepvoice-datasets/orcasound_processed_train_data.tar.gz` ?
2. Worth defining an accepted train val test split in the notebook.

File: datasets/orcasound/adapt_orcasound_dataset.ipynb [hunk 4]
```diff
@@ -0,0 +1,504 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
```
Comment (Z30G0D) [inline datasets/orcasound/adapt_orcasound_dataset.ipynb:6]: what does the file 
annotation_file = pd.read_csv('train_data/annotations.csv', sep='\t')
refers to?

**Review by Z30G0D — APPROVED** [review]
Thanks for adding the new dataset
GJ!

**Pull request merged**